src/render.ts

```diff
@@ -113,12 +113,12 @@ export function codeBlock(
  *
  * ```js
  * md.table({
- *  columns: ["Breed", "Origin", "Size", "Temperament"],
+ *  columns: ['Breed', 'Origin', 'Size', 'Temperament'],
```

Comment:
We use prettier defaults for formatting and it used `""`

Reply:
Ok, I fixed them